### PR TITLE
Improved text sizing in main menu

### PR DESCRIPTION
**Pull request by grzesiek2010**
#### What has been done to verify that this works as intended?
I've tested the fix and compared it with how it looks on master.

#### Why is this the best possible solution? Were any other approaches considered?
This is what we have discussed with @alyblenkin `textAppearanceLabelLarge` is the recommended style for this type of button according to https://github.com/material-components/material-components-android/blob/master/docs/components/Button.md#text-label-attributes and `18sp` makes it a little bit smaller.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
It doesn't require testing.

#### Do we need any specific form for testing your changes? If so, please attach one.
No.

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
No.

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

**Comment by seadowg** [comment]
@alyblenkin @grzesiek2010 how would you feel about adding our own custom text token for this? It definitely doesn't feel like we want to bump "Label large" to 18sp as this would cause problems for smaller buttons. I do imagine we'll end up using a similar style for buttons in widgets (also full width and a place we want an easy tap target vertically). I'd usually not like the idea of "extending" Material, but I personally feel OK introducing a "label extra large" (`?textAppearanceLabelExtraLarge`) token seeing as it feels like a missing part of the typography scale for us.

**Comment by alyblenkin** [comment]
@seadowg @grzesiek2010, I think adding our own custom text token makes a lot of sense because it seems like there is a gap between their typography library. And I agree that we would be using it in other similar scenarios. I also added a story for [unifying all the buttons ](https://github.com/getodk/collect/issues/5567#issue-1694879519). I still need to flesh it out more, but I think we could do them separately. Let me know what you think.

**Comment by alyblenkin** [comment]
Can I test this on my device before I add my review?

**Comment by grzesiek2010** [comment]
@seadowg something like this? https://github.com/getodk/collect/pull/5568/commits/5f4026e84e2c314f4031b1d049c4684aedfaed15

File: collect_app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <style name="TextAppearanceLabelExtraLarge" parent="TextAppearance.Material3.LabelLarge">
```
Comment (seadowg) [inline collect_app/src/main/res/values/styles.xml]: This should probably live in `typography.xml`

File: collect_app/src/main/res/values/attrs.xml [hunk 2]
```diff
@@ -18,4 +18,8 @@
     <declare-styleable name="Material3PolyFill">
         <attr name="colorSurfaceContainerLow" format="color" />
     </declare-styleable>
+
```
Comment (seadowg) [inline collect_app/src/main/res/values/attrs.xml:21]: Let's add a comment here so that it's clear these attributes are "extensions" to Material 3.

**Review by seadowg — CHANGES_REQUESTED** [review]

**Comment by seadowg** [comment]
> Can I test this on my device before I add my review?

I'll send you a build. I've got a todo to have them generated for you, but haven't had a chance to take care of that yet.

File: collect_app/src/main/res/values/styles.xml (hunk 1 above)
Comment (grzesiek2010) [reply collect_app/src/main/res/values/styles.xml]: Fixed.

File: collect_app/src/main/res/values/attrs.xml (hunk 2 above)
Comment (grzesiek2010) [reply collect_app/src/main/res/values/attrs.xml:21]: Done.

**Review by seadowg — APPROVED** [review]

**Comment by seadowg** [comment]
@alyblenkin let us know if you're happy with the changes, and we can go ahead and merge. I don't think we need QA on this one.

**Comment by alyblenkin** [comment]
Looks great!

**Pull request merged**